### PR TITLE
OPS-539 Add expiration lifecycle rules to s3

### PR DESCRIPTION
**Pull request by snovikov**
We need to set lifecycle policy for our buckets to delete old files.

### CODE REVIEW

File: s3.tf [hunk 1]
```diff
@@ -35,6 +35,21 @@ resource "aws_s3_bucket" "this" {
...
+  dynamic "lifecycle_rule" {
+    for_each = var.s3_lifecycle_rules
+    iterator = rule
+
+    content {
+      id      = rule.value.id
+      enabled = rule.value.enabled
+      prefix  = rule.value.prefix
+      expiration {
+        days = rule.value.expiration_days
```
Comment (cytopia) [inline s3.tf]: This doesn't match the example

Comment (snovikov) [reply s3.tf]: This is a simpliest way. Any suggestion for alternative?

Comment (cytopia) [reply s3.tf]: I was referring to your provided example: https://github.com/Flaconi/terraform-aws-microservice/pull/22/files#diff-ea6693f8c9ac2ab413a0b523bb824f5eR24

```hcl
    {
      id      = "tmp"
      enabled = true
      prefix  = "tmp/"
      expiration_days = 1
    }
```

Comment (cytopia) [reply s3.tf]: Why is one `expiration_days` and the other `expiration.days`?

**Pull request merged**